kv/db.go

```diff
@@ -146,3 +147,78 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", contentType)
 	w.Write(body)
 }
+
+// RegisterRPC ...
+func (s *DBServer) RegisterRPC(rpcServer *rpc.Server) error {
+	return rpcServer.RegisterName("Server", &s.rpcDBServer)
+}
+
+type rpcDBServer struct {
```

Comment:
If this type is just to create a separate method namespace for RPCs, we should probably use the same pattern as in multiraft.go: https://github.com/cockroachdb/cockroach/blob/master/multiraft/multiraft.go#L106